### ISSUE TITLE
Disable "Translate" option and/or add several spoken languages

### ISSUE THREAD

**Issue by GuillaumeLeGoc** [mastodon/mastodon#23882]
### Pitch

It would be good to be able to disable the "Translate" option that appears on toots that are not in the interface language. 

And while at it, it would be a good idea (imho) to be able to fill in languages that should not be translated.

PS : I would have thought this issue would be discussed already but I didn't find anything in the settings except the interface language, nor in the Github issues.

### Motivation

(Personal) use case : I'm French but more or less fluent in english. My whole computer and services are set to english, but I follow a lot of french-speaking folks. So my timeline is full of "Translate" button at the bottom of toots, which clutter the interface.

With that option, I could fill in that I speak both english and french and Mastodon would stop propose to translate toots that don't need to be.

**Comment by keith-v-smith** [comment]
This would address a very small gripe I have too: My interface language is set to 'English (British)' so I see 'Translate' buttons everywhere - because the posts that are in 'English' seem to be regarded as a different language.

As you might expect, clicking on 'Translate' doesn't do much in this scenario.  DeepL (at least) does't bother translating "color" into "colour" for example.

If I change the interface language to 'English' then the buttons go away (as you'd expect - so this really is a small gripe!), but I figure since the option is there, I might as well use it.

**Comment by agentx-cgn** [comment]
Number of users speaking at least two languages is constantly rising. Widespread monolingualism is only a feature of the population of huge countries, e.g. USA.

**Comment by nclm** [comment]
Yes, I wanted to open this issue as well. More people in the world are bilingual or multilingual than monolingual. We should be able to select a list of languages we don’t need translation from.

**Comment by sorz** [comment]
> My interface language is set to 'English (British)' so I see 'Translate' buttons everywhere - because the posts that are in 'English' seem to be regarded as a different language.

This, happens to all Chinese users. We have to choose one among zh-CN/zh-HK/zh-TW on the interface language, but for post, `zh` without specifier is the only choice. So the translate buttons appear in every Chinese post regardless how the interface language is set.
